app/models/user_role.rb

```diff
@@ -89,6 +91,7 @@ module Flags
 
   validates :name, presence: true, unless: :everyone?
   validates :color, format: { with: /\A#?(?:[A-F0-9]{3}){1,2}\z/i }, unless: -> { color.blank? }
+  validates :position, numericality: { greater_than_or_equal_to: -POSITION_LIMIT, less_than_or_equal_to: POSITION_LIMIT }
```

Comment:
I think there is an off-by-one, in that `POSITION_LIMIT` should be a disallowed value. But also, the form should not allow it in practice thanks to the `max` attribute.

I think this can also be more concisely and accurately written as:
```suggestion
  validates :position, numericality: { greater_than_or_equal_to: (-POSITION_LIMIT)..POSITION_LIMIT }
```

But in either case, I'm surprised `simple_form` does not output appropriate `min` and `max` values for the input. I'm also surprised there isn't a better way to do this, Rails is already aware of the allowed range, isn't there an option for it to validate against that known valid range instead of coming up with our own limits?

Reply:
> I think there is an off-by-one, in that POSITION_LIMIT should be a disallowed value. 

I had a similar worry, but I believe the added test should have caught this if it was the case.

> I think this can also be more concisely and accurately written as:

Did you mean `in` instead of `greater_than_or_equal_to`? That would be more concise indeed.

Reply:
> I had a similar worry, but I believe the added test should have caught this if it was the case.

Otoh it probably does not try to actually persist the record. Sorry, I am always a bit confused how those shoulda matchers work.

Reply:
> Did you mean `in` instead of `greater_than_or_equal_to`? That would be more concise indeed.

Yes, that's what I meant!